kde-apps/kate-addons/kate-addons-9999.ebuild

```diff
@@ -89,6 +89,8 @@ src_install() {
 
 	# provided by kde-apps/kate-lib
 	rm -v "${ED}"/usr/$(get_libdir)/libkateprivate.so.* || die
+	rm -v "${ED}"/usr/$(get_libdir)/qt6/plugins/kf6/kio/kio_kateexec.so || die
```

Comment:
Is that needed at build-time? I doubt it. Could you try disable it already at cmake stage, so it isn't built in the first place except for kate-lib?

Reply:
Yeah, this library in particular could be skipped easily.